### ISSUE TITLE
Connect-AzAccount: The term 'Connect-AzAccount' is not recognized as a name of a cmdlet, function, script file, or executable program.

### ISSUE THREAD

**Issue by prabhuwk** [Azure/azure-powershell#22222]
### Description

Recently updated Az package in the docker container and we see following error 

`PS /app> Connect-AzAccount
Connect-AzAccount: The term 'Connect-AzAccount' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS /app>`

Please help to fix issue.

### Issue script & Debug output

```PowerShell
PS /app> $DebugPreference='Continue'
PS /app> Connect-Azaccount
Connect-Azaccount: The term 'Connect-Azaccount' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS /app>
```

### Environment data

```PowerShell
PS /app> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Linux 5.15.49-linuxkit #1 SMP PREEMPT Tue Sep 13 07:51:32 UTC 2022
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

PS /app>
```

### Module versions

```PowerShell
PS /app> Get-Module Az*

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     2.12.3                Az.Accounts
Script     2.0.0                 Az.Advisor                          {Disable-AzAdvisorRecommendation, Enable-AzAdvisorRecommendation, Get-AzAdvisorConfiguration, Get-AzAdvisorRecommendation…}
```

### Error output

```PowerShell
PS /app> Resolve-AzError
Resolve-AzError: The term 'Resolve-AzError' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS /app>
```

**Comment by isra-fel** [comment]
This error usually means the "Az.Accounts" modyule was not installed or failed to be imported. From the result of " Get-Module Az*" I see it was installed, so it's probably conflicting with some other module.

Please try:
1. Start a new PowerShell session (terminal) and try again
2. If still not working, run `Import-Module Az.Accounts`. Explicitly importing the module shuold give you a detailed error.
3. If the problem still persists, run `get-module` without any parameters to get a list of modules loaded in the current session. This should give you an idea what module Az.Accounts has a conflict with.

**Comment by prabhuwk** [comment]
Please find following 
1. started new powershell and I still see same error.  
```
root@b07da63b3288:/app# pwsh
PowerShell 7.1.3
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

   A new PowerShell stable release is available: v7.3.5
   Upgrade now, or check out the release page at:
     https://aka.ms/PowerShell-Release?tag=v7.3.5

PS /app> Connect-Azaccount
Connect-Azaccount: The term 'Connect-Azaccount' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
PS /app>
```

2. Importing doesn't give any error 
```
PS /app> Import-Module Az.Accounts
PS /app>
```

3. output of `get-module` command

```
PS /app> get-module

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     2.12.3                Az.Accounts
Manifest   7.0.0.0               Microsoft.PowerShell.Management     {Add-Content, Clear-Content, Clear-Item, Clear-ItemProperty…}
Manifest   7.0.0.0               Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object…}
Script     2.1.0                 PSReadLine                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PSReadLineKeyHandler, Set-PSReadLineKeyHandler…}

PS /app>

```

Issue still persists. Please help to fix.

**Comment by isra-fel** [comment]
@prabhuwk this could also be a known issue when both AzureRM and Az are installed in the same environment. Please check out our troubleshooting guide:  https://learn.microsoft.com/en-us/powershell/azure/troubleshooting?view=azps-10.1.0#command-found-but-could-not-be-loaded

Besides, PowerShell 7.1 has reached end-of-support. Please update to at least 7.2.
